src/controllers/collection.js

```diff
@@ -25,16 +25,16 @@ const collectionSearchPOST = async (req, res) => {
     // Database read with conditional sorting based off of selected sort option from dropdown options
     switch (sort) {
     case 'alpha':
-        collections = collectionService.sortAlpha(userId, regexPattern)
+        collections = await collectionService.sortAlpha(userId, regexPattern)
```

Comment:
In the future refactor this switch case into a function with a return of collections.